packages/sanity/test/validation/dates.test.ts

```diff
@@ -0,0 +1,102 @@
+import {getFallbackLocaleSource} from '../../src/core/i18n/fallback'
+import {Rule} from '../../src/core/validation'
+
+describe('date', () => {
+  describe('with default format', () => {
+    const context: any = {client: {}, i18n: getFallbackLocaleSource(), type: {name: 'date'}}
+
+    test('min length constraint', async () => {
+      const rule = Rule.dateTime().min(Date.parse('2024-01-01'))
```

Comment:
once this is rebased with changes from `next` all the `Date.parse` can be removed and it should still work